src/container.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Container abstraction for the Vault charm."""
+
+from typing import TextIO
+
+from charms.vault_k8s.v0.vault_tls import WorkloadBase
+from ops import Container as OpsContainer
+
+
+class Container(WorkloadBase):
```

Comment:
What's the point of this class?

Reply:
This is an abstraction to unify some code between the machine and the k8s charm.